### PR TITLE
Fixes #9988: file /opt/rudder/share/relay-api/cleanup.sh is not executable, lots of cron mail about it

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://www.rudder-project.org/redmine/issues/9988

### CODE REVIEW

File: rudder-server-relay/SPECS/rudder-server-relay.spec [hunk 1]
```diff
@@ -142,7 +142,7 @@ python virtualenv/virtualenv.py flask
 
 # Get all requirements via pip
 flask/bin/pip install -r requirements.txt
-
+:x
```
Comment (amousset) [inline rudder-server-relay/SPECS/rudder-server-relay.spec]: should be removed ;)

File: rudder-server-relay/SPECS/rudder-server-relay.spec [hunk 2]
```diff
@@ -178,7 +178,7 @@ cp -r %{_sourcedir}/relay-api/flask %{buildroot}%{rudderdir}/share/relay-api/
 cp -r %{_sourcedir}/relay-api/relay_api %{buildroot}%{rudderdir}/share/relay-api/
 cp %{_sourcedir}/relay-api/apache/relay-api.wsgi %{buildroot}%{rudderdir}/share/relay-api/
 install -m 644 %{_sourcedir}/relay-api/apache/relay-api.conf %{buildroot}/etc/%{apache_vhost_dir}/relay-api.conf
-install -m 644 %{_sourcedir}/relay-api/cleanup.sh %{buildroot}%{rudderdir}/share/relay-api/
+install -m 744 %{_sourcedir}/relay-api/cleanup.sh %{buildroot}%{rudderdir}/share/relay-api/
```
Comment (amousset) [inline rudder-server-relay/SPECS/rudder-server-relay.spec]: rather use 755

**Comment by VinceMacBuche** [comment]
Commit modified

**Comment by amousset** [comment]
You can rebase and self-merge.

**Comment by VinceMacBuche** [comment]
OK, merging this PR

**Pull request merged**